sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java

```diff
@@ -1,19 +1,20 @@
 package com.microsoft.azure.mobile.crashes;
 
 import android.support.annotation.VisibleForTesting;
+import android.util.Base64;
 
-import com.microsoft.azure.mobile.crashes.model.ErrorAttachment;
-import com.microsoft.azure.mobile.crashes.model.ErrorBinaryAttachment;
+import com.microsoft.azure.mobile.crashes.ingestion.models.ErrorAttachmentLog;
 import com.microsoft.azure.mobile.utils.MobileCenterLog;
 
 import static com.microsoft.azure.mobile.MobileCenter.LOG_TAG;
 
 /**
  * Error attachment utilities.
  */
-@SuppressWarnings("WeakerAccess")
-/* TODO (getErrorAttachment): Re-enable error attachment in javadoc when the feature becomes available. Add @ before link. */
-final class ErrorAttachments {
+public final class ErrorAttachments {
+
+    public static final String CONTENT_TYPE_TEXT_PLAIN  = "text/plain";
+    public static final String CONTENT_TYPE_BINARY      = "binary";
```

Comment:
binary is not a Mime type, binary can be anything like an image or video or whatever, user will specify the type for all binaries.